docs/docs/01-ibc/03-apps/01-apps.md

```diff
@@ -32,7 +32,7 @@ module correctly corresponding to the listed steps.
 
 :::note
 
-## Pre-requisites Readings
+## Pre-requisite readings
```

Comment:
And can the linter make sure that headings use only capital letter on the first word? That would be like the best thing since sliced bread.

Reply:
I dunno man, sliced bread is pretty good if you ask me